android/engine/src/main/res/values/strings.xml

```diff
@@ -146,11 +146,32 @@
     <string name="initializing">Initializing settings &#8230;</string>
     <string name="username_sample" translatable="false">e.g JohnDoe</string>
     <string name="percentage_progress" translatable="false">%1$d%%</string>
-    <string name="error_occurred" translatable="false">Something went wrong...</string>
+    <string name="error_occurred">Something went wrong...</string>
     <string name="insights">Insights</string>
+    <string name="contact_help">"Contact help"</string>
+    <string name="offline_map">"Offline Maps"</string>
     <string name="dismiss">Dismiss</string>
+    <string name="user_info" translatable="false">User info</string>
+    <string name="assignment_info" translatable="false">Assignment info</string>
+    <string name="app_info" translatable="false">App info</string>
+    <string name="device_info" translatable="false">Device info</string>
+    <string name="refresh">Refresh</string>
     <string name="unsynced_resources">Unsynced Resources</string>
+    <string name="synced_statistics" translatable="false">Synced Statistics</string>
     <string name="all_data_synced">All data synced</string>
     <string name="invalid_offline_login_state">User set up required. Enable your internet connection </string>
-    <string name="select_month">Select Month</string>
+    <string name="select_month" translatable="false">Select Month</string>
+    <string name="user" translatable="false">User</string>
+    <string name="team" translatable="false">Team</string>
+    <string name="locality" translatable="false">Locality</string>
+    <string name="team_organization" translatable="false">Team(Organization)</string>
+    <string name="care_team" translatable="false">Care Team</string>
+    <string name="location" translatable="false">Location</string>
+    <string name="app_version_code" translatable="false">App version code</string>
+    <string name="build_date" translatable="false">Build date</string>
+    <string name="manufacture" translatable="false">Manufacture</string>
+    <string name="app_versions" translatable="false">App version</string>
+    <string name="os_version" translatable="false">OS Version</string>
+    <string name="device_date" translatable="false">Date</string>
+    <string name="device" translatable="false">Device</string>
```

Comment:
These Strings should be translatable. Delete the "translatabale = false"  property.

Reply:
Actually can you deleted this property in all the strings, only the app_name should not be translatable and any branding texts.

Reply:
Yes sure. Working on it

Reply:
also, when we remove translatable and the strings get highlighted with a red line, does it mean we have to provide the translations on the alternative string resource like that for Swahili and French? @ellykits @SebaMutuku